### ISSUE TITLE
教师模型精度还可以，学生模型精度一直很低，问题出在哪里

### ISSUE THREAD

**Issue by M-Chase** [AlibabaResearch/efficientteacher#73]
如题，在自己数据集上训练的，这种情况是需要修改教师Loss的占比权重吗？

**Comment by devyao** [comment]
try teacher_loss_weights=1.0?

**Comment by Meoooww** [comment]
大佬跑通了吗，可以加一下联系方式吗？想有偿咨询一下我为啥没跑通

**Comment by jaideep11061982** [comment]
@Meoooww  Under the custom project , we have optional Supervised training step, is it mandatory , how much data is needed in that step ?
Last step is Directly starting of SSOD training, is it the 4th step in  series of step ?what is the problem i idnt understand exactly.
Thanks in advance

**Comment by Meoooww** [comment]
> @Meoooww Under the custom project , we have optional Supervised training step, is it mandatory , how much data is needed in that step ? Last step is Directly starting of SSOD training, is it the 4th step in series of step ?what is the problem i idnt understand exactly. Thanks in advance

Sorry for the late reply. 
**my question is how to adjust the parameters in yolov5l_custon_ssod.yaml:** 
I used this model on my own dataset, and the AP of fully supervised Yolov5L is around 80, with slight overfitting. After semi-supervised on datasets of the same distribution, there is a serious overfitting problem. The rendered pseudo-label is not quite as expected (a large number of wrong labels and missing labels)

**Here are a few parameters I considered, if there are errors or omissions, please point them out:**
nms_conf_thres, nms_iou_thres:  to remove redundant prediction box
ignore_thres_low, ignore_thres_high: Corresponds to τ1, τ2 in the paper, which is used to determine whether the pseudo label is reliable
How do you regulate ignore_thres_low here, ignore_thres_high? My experiments have not yielded the desired results, it troubles me for a long time

**Comment by AlexandreDiPiazza** [comment]
Hi @Meoooww ,

I am having similar issues on some custom datasets - did you fix your problem by changing/tuning the parameters your mentioned: 
- nms_conf_thres, nms_iou_thres
- ignore_thres_low, ignore_thres_high 

Thank you :)

**Comment by Meoooww** [comment]
> Hi @Meoooww ,
> 
> I am having similar issues on some custom datasets - did you fix your problem by changing/tuning the parameters your mentioned:
> 
> * nms_conf_thres, nms_iou_thres
> * ignore_thres_low, ignore_thres_high
> 
> Thank you :)

Hi
The paper did not mention how to adjust the four parameters, so when I failed to reproduce, I guessed that these four parameters needed to be adjusted. But, I didn't succeed, I don't know what principles to follow to adjust the four parameters. If you find a suitable tuning idea, please let me know as well

(Blindly adjusting parameters has taken me too much time, and I have temporarily abandoned this project)
